server/src/utils/generate-verification-code.ts

```diff
@@ -0,0 +1,14 @@
+import crypto from "crypto";
+import moment from "moment";
+import config from "../../config.json";
+
+export default (number: string) => {
```

Comment:
Bit of a nit pick, but since the function is being used for more than phone numbers, change the name of the arg?